### ISSUE TITLE
Chime url expired after some time but it available again after 24 hours or so. Why?

### ISSUE THREAD

**Issue by abhimanusharma** [aws/amazon-chime-sdk-js#1409]
### What are you trying to do?

Trying to create unique meeting url which should not be available again once meeting has ended.
Trying to schedule a meeting in future (say 4 days in future.) that should not be accessible once the host ends the meeting or scheduled meeting time. Mute/Unmute other participants.

### How can the documentation be improved to help your use case?
How to achieve the below using aws chime sdk js?
1. https://docs.aws.amazon.com/chime/latest/ug/moderate-meeting.html Using amazon-chime-sdk-js
2. https://docs.aws.amazon.com/chime/latest/ug/chime-scheduling-calendar-app.html
3. https://docs.aws.amazon.com/chime/latest/ug/chime-organizer-call-controls.html

### What documentation have you looked at so far?

I  checked the README. and official aws docs. also checked the bug #1053 and #1044

**Comment by michhyun1** [comment]
@abhimanusharma Can you please clarify for us a few things?  Which meeting URL are you referring to? How are you scheduling meetings - the SDK does not have such functionality. Are you referring to the Chime App?

**Comment by abhimanusharma** [comment]
@michhyun1 I am referring to Chime-SDK-JS, I want a user as host who can manage other participants and moderate a meeting.
I am creating meeting using chime-sdk-js, where some urls are active after 3-4 days of meeting end or meeting creation but some meeting titles are returning not found even after 10 minutes of idle time on that meeting url. There is nothing about the expiration time of meeting url/title is mentioned anywhere in the sdk docs.

**Comment by gsudhanshu** [comment]
@abhimanusharma 

schhedule a meeting say meeting1 in your appllication for say 15th july with a host user (say user1)
then have attendees wait in a waiting room untill the host starts(this is when chime meeting is created)
have all attendees  join this meeting

manage all this in your application.

We have implemented this solution in our application.

**Comment by michhyun1** [comment]
@abhimanusharma The expiration for meetings is defined at the bottom of this page: https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-mtgs.html

**Comment by abhimanusharma** [comment]
Thank you for the clarifications.

**Comment by 3ating** [comment]
@michhyun1 I just found some rules on this [page](https://docs.aws.amazon.com/chime-sdk/latest/dg/create-mtgs.html), but I couldn't find information about "Less than two audio connections are present in the meeting for more than 30 minutes" in https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-mtgs.html. I want to know whether the documentation was not written, or if I looked in the wrong place. Thanks!
